webpages/js/metaanalyses.js

```diff
@@ -989,6 +995,8 @@
         grapeEl.setAttribute('r', getGrapeRadius(exp.wt));
         _.setAttrs(tooltipEl, '.grape', 'r', getGrapeRadius(exp.wt));
 
+        grapeEl.classList.add('c' + currentGroup%7);
```

Comment:
the number 7 needs to come from a dataset value please, so we have the number of groups and their styles in one place in the SVG template